provider/github/poster.go

```diff
@@ -95,14 +102,51 @@ func (p *Poster) postPR(ctx context.Context, e *lookout.ReviewEvent,
 		return err
 	}
 
-	_, resp, err = client.PullRequests.CreateReview(ctx, owner, repo, pr, review)
-	if err = p.handleAPIError(resp, err); err != nil {
-		return err
+	for _, req := range splitReview(review, batchReviewComments) {
+		_, resp, err = client.PullRequests.CreateReview(ctx, owner, repo, pr, req)
+		if err = p.handleAPIError(resp, err); err != nil {
+			return err
+		}
 	}
 
 	return nil
 }
 
+func splitReview(review *github.PullRequestReviewRequest, n int) []*github.PullRequestReviewRequest {
```

Comment:
I wonder, how does this function behave when `len(review.Comments) == 92`.

A small comment with a description of how comments are spliced between multiple reviews would really help here.

Reply:
not sure I understand your question. It's internal function to split 1 review to more if there are more than N comments. Isn't it obvious from the signature?
```
func splitReview(review *github.PullRequestReviewRequest, n int) []*github.PullRequestReviewRequest
```

`len(review.Comments) == 92` will behave depend on N.

Reply:
I didn't understand the question neither. I found it properly coded and the tests explained it's behaviour too.